src/app/components/node/node-link/node-link.component.html

```diff
@@ -12,7 +12,15 @@
     >
       <ng-icon [svg]="featherSearch" /> Search
     </a>
-
+    
+    <div *ngIf="false"> <!-- Temporary switch to hide the drawer button, which is still in development-->
```

Comment:
There seems to be a minor indentation issue here, are you running prettier on save in your IDE?

Reply:
Fixed (by installing prettier ;) )